src/workspace/resolver.rs

```diff
@@ -392,6 +394,108 @@ impl WorkspaceResolver {
     }
   }
 
+  /// Prepare the workspace resolver for serialization
+  ///
+  /// The most significant preparation involves converting
+  /// absolute paths into relative (based on `root_dir_url`).
+  /// It also takes care of pre-serializing non-serde internal data.
+  pub fn to_serializable(
+    &self,
+    root_dir_url: BaseUrl,
+  ) -> SerializableWorkspaceResolver {
+    SerializableWorkspaceResolver {
+      import_map: self.maybe_import_map().map(|i| {
+        SerializedWorkspaceResolverImportMap {
+          // TODO: This differs from CLI. Confirm
+          specifier: root_dir_url.make_relative_if_descendant(i.base_url()),
```

Comment:
For reviewer: this differs from CLI in the case where the base_url is not a file URL.

Reply:
@dsherret just double-checking; do you confirm this is correct?

Reply:
Yup, looks good. Sorry, I missed this question when I initially reviewed.